lib/motion-spec/mock/method_bind_unbind.rb

```diff
@@ -0,0 +1,61 @@
+# -*- encoding : utf-8 -*-
+# The meta-programming that allows us to pop methods on and off for mocking
+class Object
+  # The hidden singleton lurks behind everyone
+  def metaclass
+    class << self
+      self
+    end
+  end
+
+  def meta_eval &block
+    metaclass.instance_eval &block
+  end
+
+  # Adds methods to a metaclass
+  def meta_def name, &block
```

Comment:
Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used.<br>Use def with parentheses when there are parameters.